docs/repository_overview.rst

```diff
@@ -15,7 +15,8 @@ integration test cases. There are the following top level directories:
 
 * ``docs``: sphinx based documentation of usmqe integration tests and it's
   setup (you are reading it right now)
-* ``conf``: configuration files, only examples are commited in the repository
+* ``conf``: configuration files, only examples and `main.yaml`_ files are 
+  commited in the repository
```

Comment:
I would not consider `defaults.yaml` as a example, as it is expected to be directly used. So it might be mentioned here also.